scripts/scil_bundle_meanstd.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute mean and std along the bundle for each metric.
+Compute mean and std for the whole bundle for each metric.
```

Comment:
I really prefer it written that way! _Along_ should only be for tract profile with subdivisions